syncer/job.go

```diff
@@ -69,6 +69,7 @@ type job struct {
 	key             string
 	retry           bool
 	location        binlog.Location
+	startLocation   binlog.Location
```

Comment:
How about giving some examples of DDL/DML to explain these three locations?

Reply:
add some comment in 9ce4c9